packages/dds/tree/src/test/simple-tree/arrayNode.spec.ts

```diff
@@ -590,4 +590,28 @@ describe("ArrayNode", () => {
 			);
 		});
 	});
+
+	describe("Iteration", () => {
+		it("Concurrently iterating and editing should throw an error.", () => {
```

Comment:
When testing that something throws, you want to put as little as possible in the assert.throws block so you can test that the correct code throws the assert, and not the constructor , hydrate etc.

Reply:
Removed unnecessary code from `assert.throws` block